cfgov/ask_cfpb/tests/models/test_pages.py

```diff
@@ -828,21 +828,19 @@ def test_category_page_context_es(self):
 
     @mock.patch('ask_cfpb.models.pages.SearchQuerySet.filter')
     def test_category_page_context_no_es(self, mock_es_query):
+        blank_facets = {'answers': {},
+                        'audiences': {},
+                        'subcategories': {}}
         mock_return_value = mock.Mock()
-        mock_return_value.facet_map = json.dumps(
-            {'answers': {},
-             'audiences': {},
-             'subcategories': {}})
+        mock_return_value.facet_map = json.dumps(blank_facets)
         mock_es_query.return_value = [mock_return_value]
         mock_site = mock.Mock()
         mock_site.hostname = 'localhost'
         mock_request = HttpRequest()
         mock_request.site = mock_site
         cat_page = self.create_category_page(ask_category=self.category)
         test_context = cat_page.get_context(mock_request)
-        self.assertEqual(
-            test_context['facet_map'],
-            mock_return_value.facet_map)
+        self.assertEqual(test_context['facets'], blank_facets)
 
```

Comment:
Is there a test already for checking that redirects don't make it in the list? If not, let's add one!

Reply:
good idea!